File: 1013-fibonacci-number/fibonacci-number.cpp
class Solution {
public:
    int fib(int n) {
        if(n==0) return n;
        if(n==1) return n ;
        int i=2;
        int a=0;
        int b=1;
        while(i<=n){
            int temp =a;
            a=b;
            b = a + temp;
            i++;
        }
return b;
    }
};

File: 1013-fibonacci-number/fibonacci-number.js
/**
 * @param {number} n
 * @return {number}
 */
var fib = function(n) {
    if(n==1 || n ==0) return n;
    let a =0;
    let b=1;
    let i=2;
    while(i<=n){
        let temp=a;
        a=b;
        b = temp + b;
        i++;
    }
    return b;
};